src/transport/SecureMessageCodec.cpp

```diff
@@ -60,6 +65,11 @@ CHIP_ERROR Encrypt(Transport::SecureSession * state, PayloadHeader & payloadHead
     uint8_t * data    = msgBuf->Start();
     uint16_t totalLen = msgBuf->TotalLength();
 
+#ifdef CHIP_TRANSPORT_TRACE_ENABLED
```

Comment:
This is pretty ugly at the call site. We should be able to always "call" the macro but have it generate no code when tracing is not built.

Reply:
Done